### PR TITLE
Boxes

### PR DESCRIPTION
**Pull request by grahambancroft**
## Done

Add box styles. Update _global-settings.scss
## QA

Run ‘gulp build’ then go to the demo and have a look at the boxes

Depends on PR https://github.com/ubuntudesign/vanilla-framework/pull/311 (for the ubuntu-orange variable)
## Details

Card: https://trello.com/c/JH5ZGF5v

### CODE REVIEW

**Comment by grahambancroft** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;JH5ZGF5v&#x2F;397-add-box-styles-to-u-c-theme"> Add .box styles to u.c theme</a>

**Comment by richmccartney** [comment]
This requires #138 fixes for me to be able to test.

**Comment by grahambancroft** [comment]
This should be good to test now

**Comment by richmccartney** [comment]
Okay it builds now YAY! But the box model padding looks off now. I've attached a screenshot. Was viewed in Safari OS X.

screen shot 2016-03-09 at 09 38 57

**Comment by richmccartney** [comment]
@grahambancroft one more thing, the way we have BEM'ed on line 110 looks odd. We shouldn't do a modifier style with a block style after it.

**Comment by anthonydillon** [comment]
Thanks for this @grahambancroft. The boxes mixins are missing comments and the resource box below looks odd, difficult to see the folded corner.
vanilla framework pattern library

**Comment by barrymcgee** [comment]
@grahambancroft Is this still relevant?

File: scss/modules/_boxes.scss [hunk 1]
```diff
@@ -9,154 +9,59 @@
...
-  @include ubuntu-box-padded-feature;
   @include ubuntu-resource;
 }
 
 @mixin ubuntu-box() {
   .box {
-    @extend %box;
-    background: $white;
-    border: 1px solid $mid-grey;
+    @extend %ubuntu-box;
```
Comment (anthonydillon) [inline scss/modules/_boxes.scss]: Is there no border on `.box`?

File: scss/modules/_boxes.scss [hunk 2]
```diff
@@ -9,154 +9,59 @@
...
   }
 }
 
 @mixin ubuntu-box-grey() {
-  .box-grey {
+  .box--grey {
     @extend %box;
```
Comment (anthonydillon) [inline scss/modules/_boxes.scss]: There is no `%box` that I can see.

File: scss/modules/_boxes.scss [hunk 3]
```diff
@@ -9,154 +9,59 @@
...
-    }
-
-    .box-padded-feature {
-      .one-col {
-        float: left;
-        width: 48px;
-      }
-    }
+  .box--highlight {
+    @extend %box;
```
Comment (anthonydillon) [inline scss/modules/_boxes.scss]: Again is there `%box`?

File: scss/modules/_boxes.scss [hunk 4]
```diff
@@ -9,154 +9,59 @@
...
+    @extend %ubuntu-box;
+    @extend %ubuntu-box-highlight;
   }
 }
 
 @mixin ubuntu-resource() {
-  .resource {
+  .box--resource {
+    @extend %box;
```
Comment (anthonydillon) [inline scss/modules/_boxes.scss]: `%box`?

**Comment by deadlight** [comment]
@grahambancroft Is this change still valid? Did you action ant's comments?

**Comment by barrymcgee** [comment]
Closing as boxes are now deprecated in favour of cards upstream in Vanilla.

**Pull request closed without merging**